oak_attestation_integration_tests/README.md

```diff
@@ -0,0 +1,4 @@
+# Oak Attestation Integration Tests
```

Comment:
Could you mention the actual motivation - minimize dependencies? Otherwise it looks terrible, because why would you make a new crate for some tests of another crate.

Reply:
Do we believe that oak_attestation's test dependencies will eventually be imported to google3? If so maybe add a TODO to re-merge these crates.

Reply:
@thmsbinder excellent comment. I'll overlooked it in this PR, but added in: https://github.com/project-oak/oak/pull/4842